lib/tariff_synchronizer/mailer.rb

```diff
@@ -54,9 +54,10 @@ def file_write_error(path, reason)
       mail subject: "#{subject_prefix(:error)} Update fetch failed: cannot write update file to file system"
     end
 
-    def applied(update_names, conformance_errors = [])
+    def applied(update_names, conformance_errors, import_warnings)
```

Comment:
Hopefully `applied` is not being called anywhere else with just the first argument? Wonder why they had to use a default for the second argument?

Reply:
This is literally just for the importers (hence the apply/applied naming). I've made sure both pathways conform to the new signature